src/ImageSharp/Formats/Jpeg/PdfJsPort/Components/FastACTables.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+using System;
+using SixLabors.Memory;
+
+namespace SixLabors.ImageSharp.Formats.Jpeg.PdfJsPort.Components
+{
+    /// <summary>
+    /// The collection of lookup tables used for fast AC entropy scan decoding.
+    /// </summary>
+    internal sealed class FastACTables : IDisposable
```

Comment:
I wonder if we really need this class in it's current form. Either we should hide the `Table` property and encapsulate actual logic (initialization, retrieval of rows) inside the class, or we should drop it and define a helper method that creates  `Buffer2d<short>`.

Reply:
I like having a class but I think you're tight about the tables property. I'll hide that away and introduce an indexer.